### PR TITLE
Team3

### PR DESCRIPTION
**Pull request by nero2310**
load_gpx function

### CODE REVIEW

File: hdp/process/load.py [hunk 1]
```diff
@@ -182,14 +183,73 @@ def load_xls(xls_files: list, params: dict = {}):
...
+                    track_dict["type"] = "Trackpoint"
+                    track_dict["longitude"] = point.longitude
+                    track_dict["elevation"] = point.elevation
+                    track_dict["latitude"] = point.latitude
+                    track_dict["time"] = point.time if not "None" else None
+                    track_list.append(track_dict)
+    return track_list
+
+
+def extract_waypoints(gtx_parsed):
```
Comment (KrzysiekDev) [inline hdp/process/load.py]: Change `gtx_parsed` to `gpx_parsed`.
I was wondering if we need waypoints, but maybe we can load them.

**Review by KrzysiekDev — CHANGES_REQUESTED** [review]
You need to change typo in one place.

File: hdp/process/load.py (hunk 1 above)
Comment (nero2310) [reply hdp/process/load.py]: Oh is a misspelling, I think we should load them just in case we wanna use them.

**Review by KrzysiekDev — APPROVED** [review]
Now it's OK.

**Review by KrzysiekDev — APPROVED** [review]
Looks OK.

**Pull request merged**